### DISCUSSION TITLE
终极无敌可爱的小三月祝大家中秋快乐！！！

### DISCUSSION THREAD

**Discussion by March-777th** [share121/inter-knot#5030] [灌水]
前些天咱去罗浮的时候，跟仙舟人学了点祝福语，这边给您露一手<img src="https://bbs-static.miyoushe.com/static/2024/07/31/2dafa973b0fb29e5342403c4ead8af29_7443113916298582946.png" alt="三月七 比划" width="75" >
中秋明月照心间，愿你生活比蜜甜<img src="https://img-static.mihoyo.com/communityweb/upload/8debf4de875496c2515b005734200fe8.png" alt="三月七_吃糖" width="75" >
中秋团圆一家亲，和和美美乐无尽<img src="https://img-static.mihoyo.com/communityweb/upload/8aee1ff2c6508b43813237163cacec1c.png" alt="三月七_加油" width="75" >
中秋佳节工作顺，事业有成梦成真<img src="https://img-static.mihoyo.com/communityweb/upload/66f5c4854e9212e50fae7aeb97995a73.png" alt="三月七_开心" width="75" >
中秋友情常相伴，携手同行路不弯<img src="https://img-static.mihoyo.com/communityweb/upload/a370edf80a844b3abcececf055ab9cf9.png" alt="三月七_盯" width="75" >
中秋爱侣情更浓，甜蜜相守乐无穷<img src="https://img-static.mihoyo.com/communityweb/upload/a8b2f0d6c5173547b23b7557d9595391.png" alt="三月七_悄悄话" width="75" >
###### ~~没有的话就让咱小三月陪你吧~~
中秋游戏嗨不停，欢乐满满趣无穷<img src="https://img-static.mihoyo.com/communityweb/upload/0b5acb4c4e49bf8733806b11450f6c50.png" alt="三月七_Biu" width="75" >
 
中秋佳节至，祝福声声传。愿诸位在个人生活中幸福满满，家庭和睦团圆，工作顺利如意，友情长久坚固，爱情甜蜜美满，游戏欢乐多多<img src="https://bbs-static.miyoushe.com/static/2024/07/31/2dafa973b0fb29e5342403c4ead8af29_7443113916298582946.png" alt="三月七 比划" width="75" >

**Comment by KELERSAMA** [comment]
祝小三月永远不死 
<img src="https://bbs-static.miyoushe.com/static/2023/05/23/a2fa805a0a782cdb8ab44ddaf8d5343d_962898009752203374.png" alt="阿姬倒地了04" width="75" >
~~要是能帮我写作业就更好了~~
